stock-app/src/test/java/edu/JasonFlinkstrom/advancedjava/stock/StockServiceFactoryTest.java

```diff
@@ -0,0 +1,29 @@
+package edu.JasonFlinkstrom.advancedjava.stock;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+* Unit Test for StockServiceFactory class
+ */
+public class StockServiceFactoryTest {
+
+    /**
+    * Positive test verifies StockService is an instance of StockServiceFactory
+     */
+    @Test
+    public void createStockServicePosTest() {
+        StockServiceFactory sSF = new StockServiceFactory();
+        assertTrue(sSF.createStockService() instanceof StockService);

```

Comment:
use whole word variable names 
use the form of assert that allows you to pass in a string as the first arg. 
use this string to describe what you are asserting. 
this will make figuring out why tests fail much easier.